ramls/item.json

```diff
@@ -296,6 +296,45 @@
       "folio:linkFromField": "holdingsRecordId",
       "folio:linkToField": "id",
       "folio:includedElement": "holdingsRecords.0"
+    },
+    "lastCheckIn": {
+      "type": "object",
+      "readonly": true,
+      "additionalProperties": false,
+      "properties": {
+        "date": {
+          "type": "string",
+          "description": "Date and time of the last check in of the item."
+        },
+        "servicePointName": {
+          "type": "string",
+          "description": "Service point being used by user when item was scanned in Check in app."
+        },
+        "source": {
+          "type": "object",
+          "readonly": true,
```

Comment:
How are these properties read only? What part of the system will set them, inventory storage cannot know which user performed the check in?

Reply:
Done.